skbio/stats/composition.py

```diff
@@ -642,6 +642,112 @@ def ilr_inv(mat, basis=None, check=True):
     return clr_inv(np.dot(mat, basis))
 
 
+@experimental(as_of="0.5.1-dev")
+def alr(mat, denominator_col = 0):
```

Comment:
We'll want to have a unittest for this method.  I'd checkout the unittest for clr [here](https://github.com/biocore/scikit-bio/blob/master/skbio/stats/tests/test_composition.py#L226) for an idea about to test this.